docs/release-notes.txt

```diff
@@ -8,6 +8,128 @@
  Version 0.13
 **************
 
+Version 0.13.8
+==============
+
+**Release Date:** November 17, 2020
+
+**New Features**
+
+-  Deployment: When using ``det-deploy aws``, you can now specify the 
+   subnet that the agents should be launched into (if you are using the 
+   simple deployment type). This allows users to explicitly pick a
+   subnet/availability zone that has GPU instances.
+
+-  Security: Allow the agent to use different addresses for master 
+   certificate verification and connecting to the master.
+
+**Improvements**
+
+-  **Breaking Change:** API: Perform salting and hashing on server-side for
+   the password change endpoint. This makes this endpoint consistent with
+   the new login endpoint described at
+   https://docs.determined.ai/latest/rest-api/ .
+
+-  **Breaking Change:** Logging: Start using Fluent Bit for handling trial 
```

Comment:
What are the user-visible consequences of this change? @dzhu 

e.g., does this result in better scalability, more readable logs, more options for storing logs, ... ?

Reply:
Honestly, not much by itself (though it does come along with the filtering changes, which I'm going to add a line for separately). Do you think it's worth mentioning here how this is laying the groundwork for future things?

Reply:
Sure, I don't think that would hurt.

Reply:
Okay, how about this (appended to what's there now)?
```
This change is part of an effort to improve the handling of trial logs by increasing scalability and allowing more options for log storage.
```